pkg/plugins/resources/postgres/store.go

```diff
@@ -267,3 +301,39 @@ func (r *resourceMetaObject) GetCreationTime() time.Time {
 func (r *resourceMetaObject) GetModificationTime() time.Time {
 	return r.ModificationTime
 }
+
+func registerAndCollectMetrics(pc core_plugins.PluginContext, dbStats sql.DBStats) error {
+
+	if err := pc.Metrics().Register(postgresConnectionMetric); err != nil {
+		return err
+	}
+
+	if err := pc.Metrics().Register(postgresMaxConnectionMetric); err != nil {
+		return err
+	}
+	if err := pc.Metrics().Register(postgresClosedConnectionMetric); err != nil {
+		return err
+	}
+	if err := pc.Metrics().Register(postgresWaitConnectionMetric); err != nil {
+		return err
+	}
+	if err := pc.Metrics().Register(postgresWaitConnectionDurationMetric); err != nil {
+		return err
+	}
+
+	postgresConnectionMetric.WithLabelValues("open_connections").Set(float64(dbStats.OpenConnections))
```

Comment:
you cannot just use `.Set(float64(dbStats.OpenConnections))`. This way you will set the first value when you start the control plane and that's it. We need to follow this value. Example
```
	certGenerationsMetric := prometheus.NewGaugeFunc(prometheus.GaugeOpts{
		Help: "Number of generated certificates",
		Name: "sds_cert_generation",
	}, func() float64 {
		return float64(reconciler.GeneratedCerts())
	})
```